app/controllers/outgoing_messages/delivery_statuses_controller.rb

```diff
@@ -11,7 +11,7 @@ def show
     @show_mail_server_logs = @outgoing_message.is_owning_user?(@user)
 
     if @show_mail_server_logs
-      @mail_server_logs = @outgoing_message.mail_server_logs.to_a.map do |log|
```

Comment:
child objects are returned ad `ActiveRecord::Associations::CollectionProxy` which has Array methods like `map` and `each` so no need to convert the result